.github/workflows/lint.yml

```
@@ -28,6 +28,22 @@ jobs:
       - name: Lint
         run: composer run lint
 
+  php-cs-fixer:
+    name: php-cs check
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@master
+    - name: Set up php
+      uses: shivammathur/setup-php@master
+      with:
+        php-version: 7.4
+        coverage: none
+    - name: Install dependencies
+      run: composer i
+    - name: Run coding standards check
+      run: composer run cs:check || ( echo 'Please run `composer run cs:fix` to format your code' && exit 1 )
+
```

Spacing between steps?

```suggestion
    - name: Checkout
      uses: actions/checkout@master

    - name: Set up php
      uses: shivammathur/setup-php@master
      with:
        php-version: 7.4
        coverage: none

    - name: Install dependencies
      run: composer i

    - name: Run coding standards check
      run: composer run cs:check || ( echo 'Please run `composer run cs:fix` to format your code' && exit 1 )

```